purefa/dashboards/purefa_overview.json

```diff
@@ -0,0 +1 @@
ormance_bandwidth_bytes{dimension:mirrored_write,$ArrayName,$Env} by {fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":8,"y":7,"width":4,"height":3}},{"id":2117787392171906,"definition":{"title":"Average Top 5 Hosts by Array - Read IOPS","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"alias":"Read IOPS","formula":"top(query1, 5, 'mean', 'desc')"}],"queries":[{"query":"avg:purefa.host.performance_iops{dimension:read,$ArrayName,$Env} by {host,fa_array_name}","data_source":"metrics","name":"query1"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":10,"width":4,"height":3}},{"id":7308391345126890,"definition":{"title":"Average Top 5 Hosts by Array - Write IOPS","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"alias":"Write IOPS","formula":"top(query1, 5, 'mean', 'desc')"}],"response_format":"timeseries","queries":[{"query":"avg:purefa.host.performance_iops{dimension:write,$ArrayName,$Env} by {host,fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":10,"width":4,"height":3}},{"id":3816947770113226,"definition":{"title":"Average Top 5 Hosts by Array - Mirrored Writes IOPS","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"alias":"Mirrored Writes IOPS","formula":"top(query1, 5, 'mean', 'desc')"}],"response_format":"timeseries","queries":[{"query":"avg:purefa.host.performance_iops{dimension:mirrored_write,$ArrayName,$Env} by {host,fa_array_name}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":8,"y":10,"width":4,"height":3}}],"template_variables":[{"name":"Env","default":"*","prefix":"env","available_values":[]},{"name":"ArrayName","default":"*","prefix":"fa_array_name","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"a9w-uji-pru"}
```

Comment:
The dashboard looks like it has a lot of great info, we'd also recommend adding your logo onto the dashboard and maybe some text that summarizes the widgets available/what the integration is sending in.

Reply:
We will look into how we can do these requests.

Reply:
@anawishnoff we have updated the dashboard as suggested and added new images for the carousel